api/src/controllers/kots/KotsAPI.ts

```diff
@@ -441,7 +441,11 @@ export class KotsAPI {
           await statusServer.termination((resolve, reject, obj): boolean => {
             // Return true if completed
             if (obj.status === "running") {
-              request.app.locals.stores.kotsAppStore.setAirgapInstallStatus(obj.display_message, "running");
+              let objMsg = obj.display_message;
+              if (typeof objMsg === "string") {
```

Comment:
why not check if `objMsg` is parsable, and then create a new object if not? you won't have to update the frontend side to handle it then

Reply:
with something like
```
function isJson(value) {
    try {
      JSON.parse(value);
    } catch(err) {
      return false;
    }
  }
```